src/parsing/post.py

```diff
@@ -143,9 +143,8 @@ async def send_formatted_post(self,
                     if msg_count_new != msg_count_prev:
                         logger.debug(log_header + ', disallowed files sent as album and retrying')
                         continue
-                logger.error(log_header + f', dropped all media and retrying...')
+                logger.error(f'{log_header}, dropped all media and retrying...')
```

Comment:
Either make all strings that start with `log_header` are f-string, or do the opposite.

Reply:
All done.

Reply:
Have you pushed your new commit(s)?